### PR TITLE
Print SSH upstream in brackets to distinguish port properly

### PR DESCRIPTION
**Pull request by ton31337**
Before:
```
% sshpiperd pipe list
donatas -> root@2a02:4231:face:1001:1610:931c:2980:9d9d:22
```

After:
```
% sshpiperd pipe list
donatas -> root@[2a02:4231:face:1001:1610:931c:2980:9d9d]:22
```

### CODE REVIEW

File: sshpiperd/utils/common.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+)
+
+func FormatIPAddress(host string) string {
+	ip4 := net.ParseIP(host).To4()
+
+	if ip4 != nil || !strings.Contains(host, ":") {
+		return host
+	}
+
+	return fmt.Sprintf("[%s]", host)
```
Comment (tg123) [inline sshpiperd/utils/common.go:16]: do you think we need to verify if it is IPv6 here?

Comment (ton31337) [reply sshpiperd/utils/common.go:16]: It's already verified above against ipv4 and `:` or what do you mean? :)

Comment (tg123) [reply sshpiperd/utils/common.go:16]: `FormatIPAddress("127.0.0.1:22")` === `[127.0.0.1:22]`
is it correct?

Comment (ton31337) [reply sshpiperd/utils/common.go:16]:
```
addr := fmt.Sprintf("%v:%v", utils.FormatIPAddress(host), port)
```
and
```
Host:             utils.FormatIPAddress(host),
Port:             port,
```
Is it possible to have `<host:port, port>` pair in the code as in above examples?

Comment (tg123) [reply sshpiperd/utils/common.go:16]: ah you are right 🤣

Comment (ton31337) [reply sshpiperd/utils/common.go:16]: if it's possible, then yes, it needs to be fixed :smile:

**Pull request merged**